### PR TITLE
Add to compass task generated images path

### PR DESCRIPTION
**Pull request by felquis**

### CODE REVIEW

**Comment by felquis** [comment]
@passy I added `<%%=`.

**Comment by passy** [comment]
Is `generatedImagesDir` the directory where generated images are written to? The `app` directory should never be touched by a grunt task. That should go into `.tmp` then.

**Comment by felquis** [comment]
`generatedImagesDir` is where the generated images of compass(e.g sprite generator) will be saved.

Currently the webapp are saving these generated images in app/images..

**Comment by passy** [comment]
Would be nice if we could move that to `.tmp` together with the compiled CSS.

**Comment by passy** [comment]
@felquis I tried this quickly by generating a sprite from a fresh `yo webapp` scaffold and the files are correctly sitting in `.tmp/images/generated`. The path in the CSS file, however, is still `/images/path-to-image.png`. I think the http path to the generated images still needs to be adjusted.

**Comment by sindresorhus** [comment]
Can we get this resolved?

**Comment by felquis** [comment]
@sindresorhus I will make these corrections tomorrow. ;D Holiday :beers: 

The issue #54 closes #43 and #35?

**Comment by sindresorhus** [comment]
Enjoy :D

**Comment by addyosmani** [comment]
Enjoy the holiday! Thanks for making time to address when they're over :)

**Comment by felquis** [comment]
@passy @sindresorhus Guys, now ["Every little thing"](http://j.mp/11Du03v) will works fine :+1:

**Comment by felquis** [comment]
Just a note: When we use sprites, the folder that contains the images of the sprite, also is copied to the `dist` directory. I don't think that this is bad. But we can use a name convertion to don't copy images of sprite to the `dist` directory, or something more smartest.

**Comment by passy** [comment]
Thanks for the update @felquis, will try it later today. Could you fix the merge conflict?

**Comment by felquis** [comment]
@passy I'm getting an error, using the current branch `master` webapp 
yo-error

**Comment by passy** [comment]
You may need to manually update the yeoman-generator version to the git
master.

**Comment by felquis** [comment]
@passy tell me if something is wrong in this pull request :)

File: app/templates/Gruntfile.js [hunk 1]
```diff
@@ -287,6 +288,13 @@ module.exports = function (grunt) {
                         'images/{,*/}*.{webp,gif}',
                         'styles/fonts/*'
                     ]
+                },{
```
Comment (passy) [inline app/templates/Gruntfile.js]: Space after the comma, please. :)

**Comment by felquis** [comment]
Done o/

**Comment by sindresorhus** [comment]
Thanks @felquis

**Pull request merged**